goodtablesio/integrations/github/models/repo.py

```diff
@@ -7,6 +7,13 @@ class GithubRepo(Source):
         'polymorphic_identity': 'github'
     }
 
+    def to_api(self):
```

Comment:
I guess the point of this and not use `to_json()` is to control exactly what gets sent, +1